https://issues.redhat.com/browse/ACM-14244 suggestion (#7250)
open-cluster-management/rhacm-docs: release_notes/known_issues_governance.adoc

[#policy-status-repeated-updates]
== Policy status shows repeated updates when enforced

> If a policy is set to `remediationAction: enforce` and is repeatedly updated, the {acm-short} console shows repeated violations with successful updates. The repeated updates produces many policy events, which can cause the `governance-policy-framework-addon` pod to run out of memory and crash. See the following two possible causes and solutions for the error:

Reviewer: _Repeated updates produce multiple policy events, which can cause..._ 

Sentence was a bit odd for me -- just a suggested to shorten--_multiple_ used here because of "repeat" but feel free to can it.

Author: Great suggestion, I updated the statement.

> If a policy is set to `remediationAction: enforce` and is repeatedly updated, the {acm-short} console shows repeated violations with successful updates. The repeated updates produces many policy events, which can cause the `governance-policy-framework-addon` pod to run out of memory and crash. See the following two possible causes and solutions for the error:

Author: ```suggestion
If a policy is set to `remediationAction: enforce` and is repeatedly updated, the {acm-short} console shows repeated violations with successful updates. Repeated updates produce multiple policy events, which can cause the `governance-policy-framework-addon` pod to run out of memory and crash. See the following two possible causes and solutions for the error:
```

+
To resolve the issue, disable the policy and compare the differences between `objectDefinition` in the policy and the object on the managed cluster. If the keys are different or missing, Kubernetes might have processed the keys before applying them to the object, such as removing keys containing default or empty values.

> - When you run out of memory because there are too many policy events created from continuous compliance changes, the `governance-policy-framework-addon` pod crashes.

Reviewer 2: Could you reword this so it's a little more clear the Out of Memory issue is specific to the governance policy framework addon pod as opposed to other concerns with the events?

> - When you run out of memory because there are too many policy events created from continuous compliance changes, the `governance-policy-framework-addon` pod crashes.

Author: @gparvin here is my attempt. I just want to be clear that we are deciding to move forward with the troubleshoot topic or only list the issue as a cause. But not to document both. However I did still update the previous PR.
```suggestion
- When you run out of memory on the `governance-policy-framework-addon` pod because there are too many policy events created from continuous compliance changes, the pod crashes.
```

Reviewer 2: Can this be simplified by just saying, "The repeated updates produces many policy events which can cause the `governance-policy-framework-addon` pod to run out of memory and crash."?

Author: ```suggestion
- The repeated updates produces many policy events, which can cause the `governance-policy-framework-addon` pod to run out of memory and crash.
```

> //should this bullet point be apart of the second point?| MJ | 11/18/24

Author: ```suggestion
```
